### ISSUE TITLE
waitForViewWithAccessibilityLabel(label:value:traits) with UISlider fails when it should pass

### ISSUE THREAD

**Issue by albertbori** [kif-framework/KIF#565]
The following test:

```
tester.setValue(1.0, forSliderWithAccessibilityLabel: "Amount") //Slider range is: 0-4
tester.waitForViewWithAccessibilityLabel("Amount", value: "25", traits: UIAccessibilityTraitNone)
```

Throws the following failure message:

> The step timed out after 10.00 seconds: Found an accessibility element with the label "Amount", but with the value "25", not "25"

Workaround:

```
var slider = tester.waitForViewWithAccessibilityLabel("Amount") as? UISlider
XCTAssertNotNil(slider, "Amount Slider was nil")
if let slider = slider {
    var value: Float = 1.0
    XCTAssertEqual(slider.value, value, "Incorrect Amount Slider value")
}
```

**Comment by phatmann** [comment]
Can you walk through the call to see why it fails?

**Comment by albertbori** [comment]
I only know Swift, but I'll give it a shot when I get some time.
